app/files/entrypoints/19-php_ini.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+TEMPLATE=/usr/local/etc/php/conf.d/docker-php-ext-zzz-dynamic.ini.ep
+TARGET=/usr/local/etc/php/conf.d/docker-php-ext-zzz-dynamic.ini
+
+if [ -f "$TEMPLATE" ]; then
+  echo "Prepare PHP docker-php-ext-zzz-dynamic.ini conf..."
+
+  doas ep -v "$TEMPLATE"
```

Comment:
This seems to fail with `app:php-8.2` to:

```
boot :: /srv/http/drupal  % docker compose logs app
core-app  | # Source /entrypoints/00-umask.sh
core-app  | # Source /entrypoints/10-ssh-agent.sh
core-app  | No socket found
core-app  | # Source /entrypoints/15-xdebug.sh
core-app  | Start with Xdebug enabled. Remove XDEBUG_ENABLE=true ENV variable to disable it.
core-app  | Already enabled...
core-app  | # Source /entrypoints/19-php_ini.sh
core-app  | Prepare PHP docker-php-ext-zzz-dynamic.ini conf...
core-app  | doas: ep: Exec format error
```

Reply:
Could it be a Linux vs Mac thing as I don't get that error 🤔